pkg/config/config.go

```diff
@@ -1 +1,22 @@
 package config
+
+type DaemonConfig struct {
+	PeriodicUpdate int `json:"periodic_update"`
+	GuidPool       GuidPoolConfig
+	SubnetManager  SubnetManagerPluginConfig
+}
+
+type GuidPoolConfig struct {
+	GuidRangeStart string `json:"guid_range_start"`
+	GuidRangeEnd   string `json:"guid_range_end"`
+}
+
+type SubnetManagerPluginConfig struct {
+	Name        string `json:"subnet_manager_name"`
+	Username    string `json:"subnet_manager_username"`
+	Password    string `json:"subnet_manager_password"`
+	Address     string `json:"subnet_manager_address"`
+	Port        int    `json:"subnet_manager_port"`
+	HttpSchema  string `json:"subnet_manager_httpSchema"`
+	Certificate string `json:"subnet_manager_certificate"`
```

Comment:
what we expect here? can we had comment?

Reply:
added